flask-app/requirements.txt

```diff
@@ -0,0 +1,10 @@
+connexion[swagger-ui] >= 2.6.0; python_version>="3.6"
```

Comment:
Why is there two requirements.txt? We should keep only one.

Reply:
one was stock generated for python 3.5 and 3.6 . Does not work for 3.7+ so I added another . If we don't want backwards compatibility I'll remove one.

Reply:
Let's keep only one `requirements.txt` for 3.7. Anyone can easily set up a python environment with 3.7.